src/main/java/com/mcmoddev/alt/api/PluginLoader.java

```diff
@@ -37,18 +42,19 @@ ResourceLocation getResourceLocation() {
 
 	private final List<PluginData> dataStore = new ArrayList<>();
 
-	public void load() {	
-		ModList.get().forEachModContainer((modid, container) -> {
-			Object modInstance = container.getMod();
-			Class<?> modClass = modInstance.getClass();
-			ALTPlugin annotation = modClass.getAnnotation(ALTPlugin.class);
-			if (annotation != null) {
-				final String modId = annotation.modid();
-				final String resourceBase = annotation.resourcePath();
-				PluginData pd = new PluginData( modId, resourceBase);
-				dataStore.add(pd);
-			}
-		});
+	public void load() {

```

Comment:
...I'm going to have to see if this can be easily backported as well - for ALT, MMD OreSpawn and MMDLib. This is much more compact and a lot nicer than the existing code :)